lib/account/pages/account_page.dart

```diff
@@ -15,11 +20,68 @@ class _AccountPageState extends State<AccountPage> {
     supabaseClient.auth.signOut();
   }
 
+  void showDesignDialog(BuildContext context) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) => StatefulBuilder(
+        builder: (context, setState) => AlertDialog(
+          content: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: List.generate(
+              ThemeMode.values.length,
+              (index) => RadioListTile(
+                  title: Text(
+                    ThemeMode.values[index].getName(context),
+                  ),
+                  value: ThemeMode.values[index],
+                  groupValue: themeManager.currentThemeMode,
+                  onChanged: (ThemeMode? value) {
+                    setState(() {
+                      themeManager.setTheme(value);
+                    });
```

Comment:
```suggestion
                    });
                    Navigator.of(context).pop();
```

Reply:
@Eddie-42 what do you think? I don't know what is best here (or what is most standard in Android Apps)

Of course it's easy to leave the dialog by just tapping anywhere else.

Reply:
For a11y, there will be an "OK" button at the bottom because people using Screen Reader aren't able to click outside the Alert. I initially removed it because it adds A LOT more space at the bottom and that looks very weird because half of the window isn't filled by radio options, but I do think it's also standard to have an OK button

Reply:
But yeah this is already implemented in the a11y branch. I will create a pull request after this is merged I think because it is based on this branch

Reply:
Okay so I'll resolve the conversation and this will be fixed with the next PR